examples/jsm/nodes/functions/PhysicalLightingModel.js

```diff
@@ -40,7 +41,9 @@ const RE_IndirectSpecular_Physical = new ShaderNode( ( inputs ) => {
 
 	computeMultiscattering( singleScattering, multiScattering );
 
-	const diffuse = diffuseColor.mul( singleScattering.add( multiScattering ).oneMinus() );
+	const totalScattering = singleScattering.add( multiScattering );
+
+	const diffuse = diffuseColor.mul( max( totalScattering.r, max( totalScattering.g, totalScattering.b ) ).oneMinus() );
```

Comment:
```suggestion
	const diffuse = diffuseColor.mul( max( totalScattering.r, totalScattering.g, totalScattering.b ).oneMinus() );
```

Reply:
It was the first thing I did ( even deserves an update soon ), but I think we just added it to `OperatorNode`. I updated using chain approach for now.